### ISSUE TITLE
это же note после re?

### ISSUE THREAD

**Issue by solanum-tuberosum** [abaevdict/abaevdict-tei#40]
![image](https://user-images.githubusercontent.com/13102259/161754753-dd41c114-dbc4-4032-8c8b-74fa61c912fc.png)
сделала вот так, нормально?
![image](https://user-images.githubusercontent.com/13102259/161754788-4f57130e-cb53-4fb2-8797-28228788a8ad.png)

**Comment by obelyaev** [comment]
Да, нормально. Немного смущает только, что ryxæn не записан как вариант основной лексемы. М.б. добавить его? Или лучше не менять замысел Абаева?
